pkg/api/meta/interfaces.go

```diff
@@ -144,6 +144,7 @@ type RESTMapping struct {
 // consumers of Kubernetes compatible REST APIs as defined in docs/api-conventions.md.
 type RESTMapper interface {
 	VersionAndKindForResource(resource string) (defaultVersion, kind string, err error)
```

Comment:
Done, they are not unique right now, but that is because multiple apigroup support is not complete. Clarified in comments.